willa/tind/format_tind_context.py

```diff
@@ -0,0 +1,65 @@
+"""
+Format Tind fields for context in Chat response
+"""
+
+DISPLAY_MAPPINGS: dict = {
+    'tind_id': 'Tind ID:',
+    'isPartOf': 'Project Name:',
+    'title': 'Title:',
+    'contributor': 'Contributor:'
+}
+
+
+def get_tind_url(tind_id: str) -> str:
+    """Create a Tind link using the Tind ID
+
+    :param: tind_id
+    :returns: URL to Tind record
+    """
+
+    tind_url = f"https://digicoll.lib.berkeley.edu/record/{tind_id}"
+    return tind_url
+
+
+def get_tind_context(docs: list) -> str:
+    """Provide formatted Tind metadata for chatbot responses.
+
+    :param: Array of documents from chatbot 
+    :returns: Formatted text of select Tind data
+    """
+
+    tind_data = ''
+    tind_ids = {}
+    for doc in docs:
+        tind_id = doc.metadata['tind_metadata']['tind_id']
+        if tind_id in tind_ids:
+            continue
+
+        tind_data += f"\n\n{process_fields(doc.metadata['tind_metadata'])}\n___________\n\n"
+        tind_ids[tind_id] = tind_id
+
+    return tind_data
+
+
+def process_fields(tind_rec: dict) -> str:
+    """Provide formatted select Tind fields to include in Chat response
+    
+    :param: dict of Tind metadata
+    :returns: A formatted string to be included in chat display
+    """
+
+    fields = ['tind_id', 'title', 'contributor', 'isPartOf']
+
+    formatted_str = ''
+    for field in fields:
+        if tind_rec[field] is not None and isinstance(tind_rec[field], list):
+            for value in tind_rec[field]:
+                formatted_str += f"{DISPLAY_MAPPINGS[field]} {value}\n"
```

Comment:
i think this might be able to be simplified: 
```suggestion
        if isinstance(tind_rec.get(field), list):
            formatted_str += [f"{DISPLAY_MAPPINGS[field]} {value}" for value in tind_rec[field]].join('\n')
```